dotcom-rendering/src/components/SeriesSectionLink.tsx

```diff
@@ -198,16 +198,17 @@ export const SeriesSectionLink = ({
 		? themePalette('--series-title-match-text')
 		: themePalette('--series-title-text');
 
-	if (inTagLinkTest) {
+	if (shouldShowTagLink) {
 		return (
 			<TagLink
-				format={format}
-				sectionLabel={'Euro 2024'}
-				sectionUrl={'football/euro-2024'}
+				sectionUrl="sport/olympic-games-2024"
+				sectionLabel="Paris Olympic Games 2024"
```

Comment:
Still hardcoding this as we are currently only interested in showing on Olympics pages, similar to the Euros before